build.sbt

```diff
@@ -161,6 +162,11 @@ packageSynapseML := {
          |    ],
          |    zip_safe=True,
          |    package_data={"synapseml": ["../LICENSE.txt", "../README.txt"]},
+         |    extras_require={
+         |      "required": [
+         |        "recommenders==1.0.0",
```

Comment:
do we need this dep if we also dep on sarplus by markus? i think recommenders I pretty large

Reply:
let me confirm... but i believe recommenders has the PySpark bindings.

Is the concern recommenders? or its deps? I can reach out to reco about having a "recommenders[sarplus]" extras so we only get the exact python deps that we need?